_posts/en/newsletters/2020-12-16-newsletter.md

```diff
@@ -0,0 +1,331 @@
+        correct.
+
+- **New PSBT version proposed:** Andrew Chow, author of the [BIP174][]
+  specification of [Partially Signed Bitcoin Transactions][topic psbt]
+  (PSBTs), [proposed][psbt2] a new version of PSBTs that will contain
+  several backwards incompatible features, although it'll be largely the
+  same as the current version 0 PSBT.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Bitcoin Wallet Tracker adds descriptor support:**
+  Bitcoin Wallet Tracker's [0.2.0 release][bwt 0.2.0] adds support for tracking
+  [output script descriptors][topic descriptors] and introduces [libbwt][libbwt github],
+  a library for allowing Electrum-backed wallets to easily support a Bitcoin
+  Core full node.
+
+- **JoinMarket now defaults to native segwit addresses:**
+  While JoinMarket supported segwit since 0.5.1, [version 0.8.0][joinmarket 0.8.0]
+  now uses bech32 native segwit addresses by default for [coinjoins][topic coinjoin].
+
+- **Bisq adds segwit for trade transactions:**
+  Building on [previous bech32 support][news120 bisq segwit] for deposits and
+  withdrawals, [Bisq v1.5.0][bisq bech32 blog] adds segwit support within trade
+  transactions as well as implementing fee optimizations.
+
+- **PSBT Toolkit v0.1.2 released:**
+  [PSBT Toolkit][psbt toolkit github], software that "aims to give you a nice
+  gui that gives you functionality for PSBT interactions", released various
+  improvements in its 0.1.2 version.
+
+- **Sparrow adds Replace-By-Fee:**
+  [Sparrow 0.9.8][sparrow 0.9.8] adds [Replace-By-Fee (RBF)][topic rbf]
+  functionality and support for [HWI][topic hwi] 1.2.1.
+
+- **Ledger Live adds Bitcoin Core full node support:**
+  Ledger Live, using the open source [Ledger StackSat][satstack github]
+  application, can now [connect to a Bitcoin full node][ledger full node] for
+  sending transactions and providing balances in a more private way, without
+  using Ledger’s [explorers][topic block explorers].
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What is the difference between "native segwit" and "bech32"?]({{bse}}100434)
+  Murch describes that bech32 is the address format for encoding native segwit
+  v0 outputs, while, due to a [length extension mutation][news78 bech32 mutability],
+  a modified bech32 address format, "bech32m", will encode Pay-to-Taproot (P2TR)
+  segwit v1 outputs.
```

Comment:
This second part "while, due to .... segwit v1 outputs." isn't a direct answer to the question "what's the difference between 'native segwit' and 'bech32'?" and is provided in the stackexchange answer for more detail/context. I think it should be omitted from this summary, or at least summarised in a separate sentence: "Murch goes on to explain that due to a ..." or similar.

Reply:
Agreed. Amended to keep the second part in as I think it’s the more interesting piece here.